.dockerignore

```diff
@@ -19,6 +23,5 @@ run
 .env.local
 
 # Node is only needed for building, not running
-package.json
 node_modules
-yarn.lock
+Gemfile.lock
```

Comment:
Remove `Gemfile.lock`. We need lock files in the container when we install dependencies.